src/hooks-log.coffee

```diff
@@ -0,0 +1,23 @@
+util = require 'util'
+
+hooksLog = (logs = [], logger, logVariant, content) ->
+  if arguments.length is 4 and logVariant in ['info', 'debug', 'warn', 'verbose', 'error', 'log']
```

Comment:
I can do it as if `hooks.log` would behave as calling `logger.info some, things, that, are, long` (as it works for winston - our logging platform).

But I would really appreciate to use just `hooks.log` as simply as possibly. So no, it is not the same as `console.log` since it obeys the used log-level.